### DISCUSSION TITLE
"404 You are logged in" error

### DISCUSSION THREAD

**Discussion by Awesomedonut** [vercel/community#5485] [Account Issue]
### Question

I want to deploy a github repo but it just shows me a "404 you are logged in as myemail@gmail.com" error whenever I go on the vercel website. I tried going to my dashboard, https://vercel.com/new, and other places on the vercel web app but I just got the aforementioned 404.

**Comment by swarnava** [comment]
Hi,


Do you have a screencast which we can look into?

**Reply by Awesomedonut** [reply]
Here is what I see:

<img width="1338" alt="Screenshot 2024-01-16 at 11 36 31 AM" src="https://github.com/vercel/community/assets/12404874/1dbd63eb-049c-4b10-a87a-dd095ca2b2a6">

If I try to go to any other page, it shows the same:
<img width="1317" alt="Screenshot 2024-01-16 at 11 37 27 AM" src="https://github.com/vercel/community/assets/12404874/b8825d93-bd54-4b67-8e68-48a6fe001f38">
<img width="1313" alt="Screenshot 2024-01-16 at 11 37 35 AM" src="https://github.com/vercel/community/assets/12404874/3fdb632b-72af-4e84-b20c-fd529e033162">

**Reply by swarnava** [reply]
Can you try login now and let us know if you are still experiencing this?

**Reply by Awesomedonut** [reply]
<img width="1307" alt="Screenshot 2024-01-18 at 8 59 49 PM" src="https://github.com/vercel/community/assets/12404874/8ac028cb-5e89-49e5-92c0-9f86bcd7f88b">

I logged in now, thank you :D 
But I don't see the "hobby" label and it just directs me to this page. I don't want to make a team; I just want to host a project.

**Reply by swarnava** [reply]
We have deprecated Hobby Personal account recently. You can create a team and keep yourself as Member which will just like your Hobby account.

**Reply by Awesomedonut** [reply]
It only shows the pro options for me
<img width="1209" alt="Screenshot 2024-01-19 at 5 44 26 PM" src="https://github.com/vercel/community/assets/12404874/d59bab43-3f64-49d9-8592-399f17201416">

<img width="1234" alt="Screenshot 2024-01-19 at 5 45 41 PM" src="https://github.com/vercel/community/assets/12404874/c678f51b-a981-48e8-bd1c-d0e5ac3f1d90">


Is there a free option available, like the hobby personal account?
